app/models/vote.rb

```diff
@@ -0,0 +1,4 @@
+class Vote < ActiveRecord::Base
+  validates :voter_id, :candidate_id, presence: true, numericality: { only_integer: true }
```

Comment:
Again, let's not validate `voter_id` and `candidate_id`. We'll have to figure out how we do this validation since both of these are users, and I doubt we can do `validates :user, :user`. We don't need `numericality`; Rails will handle this on the backend.